spring-boot-2-demo-app/src/test/java/io/github/mfvanek/spring/boot2/test/controllers/TimeControllerTest.java

```diff
@@ -37,6 +39,7 @@
 
 @ExtendWith(OutputCaptureExtension.class)
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@AutoConfigureWebTestClient(timeout = "10000")
```

Comment:
I thinks it's redundant

Reply:
TimeControllerTest fails without this because default timeout on webclient is not enough to pass tests up to the end

Reply:
Wrong test. You have to setup mock for http://localhost:8080 or configure proper timeouts for http call to that endpoint.